### PR TITLE
setstatus: clear up the wording around documenting descriptions

### PR DESCRIPTION
**Pull request by tsloughter**
## Changes

I believe this change better describes what the spec is trying to say about documentation for error status code. The wording is confusing because it refers to what I believe is the `Description` as a "status code".

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

File: specification/trace/api.md [hunk 1]
```diff
@@ -574,11 +574,12 @@ The status code SHOULD remain unset, except for the following circumstances:
...
 An attempt to set value `Unset` SHOULD be ignored.
 
-When the status is set to `Error` by Instrumentation Libraries, the status codes
+When the status is set to `Error` by Instrumentation Libraries, the `Description`
 SHOULD be documented and predictable. The status code should only be set to `Error`
 according to the rules defined within the semantic conventions. For operations
 not covered by the semantic conventions, Instrumentation Libraries SHOULD
-publish their own conventions, including status codes.
+publish their own conventions, including all possible values of `Description`
+and what they mean.
```
Comment (bogdandrutu) [inline specification/trace/api.md:582]: Does it mean that "Description" is a finite set of values? Low cardinality?

**Comment by bogdandrutu** [comment]
Ping @tsloughter

File: specification/trace/api.md (hunk 1 above)
Comment (tsloughter) [reply specification/trace/api.md:582]: Sorry, forgot to reply to this. 

Yes, it does, and is what the spec already defined.

Comment (bogdandrutu) [reply specification/trace/api.md:582]: Not sure if this was the intent. But I am happy with this.

**Comment by carlosalberto** [comment]
Ping @tsloughter ;)

**Comment by tsloughter** [comment]
@carlosalberto I don't think there is any more I can do :). I did just do the update branch, but aside from that are there any remaining questions I missed?

**Comment by tsloughter** [comment]
Maybe the github action that marks as stale and then closes should instead email approvers who haven't reviewed yet if there are already 2 +1's on a PR?

**Comment by carlosalberto** [comment]
I was triple checking this, and my impression is that this text is a left over of the time when we had multiple status code (gRPC style). This PR seems overall good to me, with the exception of 

> including all possible values of `Description` and what they mean.

as I'd personally remove the "all", in order to signal that, even if we want to document all possible cases, a) library authors can, _if needed_, put whatever they need there - and b) it's _technically_ not a low cardinality value.

(Overall I remember seeing the actual Exception/Error message in the description, e.g. https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L904)

**Review by bogdandrutu — APPROVED** [review]

**Comment by tsloughter** [comment]
I think I'd be fine with removing "all". Those who have already approved, @bogdandrutu @jmacd what do you think?

**Comment by reyang** [comment]
+1 on removing `all`

**Review by reyang — APPROVED** [review]

File: specification/trace/api.md [hunk 2]
```diff
@@ -576,11 +576,12 @@ The status code SHOULD remain unset, except for the following circumstances:
 
 An attempt to set value `Unset` SHOULD be ignored.
 
-When the status is set to `Error` by Instrumentation Libraries, the status codes
+When the status is set to `Error` by Instrumentation Libraries, the `Description`
 SHOULD be documented and predictable. The status code should only be set to `Error`
 according to the rules defined within the semantic conventions. For operations
 not covered by the semantic conventions, Instrumentation Libraries SHOULD
-publish their own conventions, including status codes.
+publish their own conventions, including all possible values of `Description`
```
Comment (arminru) [inline specification/trace/api.md:583]:
```suggestion
publish their own conventions, including possible values of `Description`
```
As suggested in the main conversation.

File: specification/trace/api.md [hunk 3]
```diff
@@ -576,11 +576,12 @@ The status code SHOULD remain unset, except for the following circumstances:
...
+and what they mean.
```
Comment (arminru) [inline specification/trace/api.md:584]: Please mention this refinement in the changelog to make instrumentation authors who follow the spec releases aware of it.

**Review by arminru — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

File: specification/trace/api.md (hunk 2 above)
Comment (bogdandrutu) [reply specification/trace/api.md:583]: +1

**Review by carlosalberto — APPROVED** [review]

**Review by alanwest — APPROVED** [review]

**Pull request merged**